combat/CombatSystem.cpp

```diff
@@ -744,13 +784,15 @@ namespace {
         if (available_fighters > 0 && !part_fighter_launch_capacities.empty()) {
             for (auto& launch : part_fighter_launch_capacities) {
                 int to_launch = std::min(launch.second, available_fighters);
-
+                std::string part_name = launch.first;
```

Comment:
probably `const std::string&`

Reply:
actually that local is unused, will remove